packages/amplication-client/src/Resource/ResourceHome.tsx

```diff
@@ -49,12 +59,19 @@ const ResourceHome = ({ match, innerRoutes }: Props) => {
               backgroundColor:
                 resourceThemeMap[currentResource?.resourceType].color,
             }}
+            onMouseEnter={handleMouseEnter}
+            onMouseLeave={handleMouseLeave}
           >
             <ResourceNameField
+              hovered={hovered}
+              hoverHandler={setHovered}
               currentResource={currentResource}
               resourceId={resourceId}
             />
-            <div className="circle-badge-container">
+            <div
+              className="circle-badge-container"
+              style={{ border: "1px solid red" }}
```

Comment:
why inline style and not part of the css

Reply:
Yes, I've removed this style, it was just for debugging purpose